### PR TITLE
Fix tests: actually compare against fixtures

### PR DESCRIPTION
**Pull request by f-f**
/cc @JordanMartinez

### CODE REVIEW

**Comment by JordanMartinez** [comment]
Ha, literally wrote something similar just now only to see that you've already addressed it.

File: test/fixtures/test-output.txt [hunk 1]
```diff
@@ -1,9 +1,5 @@
 🍝
 You should add some tests.
 Installing 3 dependencies.
-"prelude" already installed
-"console" already installed
-"effect" already installed
 Installation complete.
 Build succeeded.
-Tests succeeded.
```
Comment (JordanMartinez) [inline test/fixtures/test-output.txt]: I don't think this was supposed to be removed... At least, it would fail the test on mine if it did.

Comment (JordanMartinez) [reply test/fixtures/test-output.txt]: Yup
```
Output doesn't match fixture!


---- Expected ------
🍝
You should add some tests.
Installing 3 dependencies.
"prelude" already installed
"console" already installed
"effect" already installed
Installation complete.
Build succeeded.
Tests succeeded.

---- Actual --------
🍝
You should add some tests.
Installing 3 dependencies.
Installation complete.
Build succeeded.
Tests succeeded.

-----------
```

Comment (f-f) [reply test/fixtures/test-output.txt]: Sorry it slipped in :)

**Pull request merged**